test/functional/mempool_accept.py

```diff
@@ -333,6 +340,45 @@ def run_test(self):
             maxfeerate=0,
         )
 
+        # Prep for tiny-tx tests with wsh(OP_TRUE) output
+        best_block = self.nodes[0].getbestblockhash()
+        best_block_time = node.getblock(best_block)['time']
+        block_time = best_block_time + 1
+        tip = int(node.getbestblockhash(), 16)
+        coinbase = create_coinbase(node.getblockcount()+1)
+        # Need an output that allows a tiny non-witness spend with standard relay
+        # which raw OP_TRUE fails
+        coinbase.vout[0].scriptPubKey = script_to_p2wsh_script(CScript([OP_TRUE]))
+        coinbase.rehash()
+        block = create_block(tip, coinbase, block_time)
+        block.solve()
+        node.submitblock(block.serialize().hex())
+        self.generate(node, 100)
```

Comment:
Seems overkill to mine 100 blocks, when you can simply create the output with the miniwallet?

Reply:
I went down the rabbit hole with naked OP_TRUE, forgetting that wsh(OP_TRUE) is standard for making and spending... will remove